BeVolt/App/inc/Voltage.h

```diff
@@ -35,6 +35,23 @@ SafetyStatus Voltage_IsSafe(void);
  * @return pointer to index of modules that are in danger
  */
 uint16_t *Voltage_GetModulesInDanger(void);
+ 
+/** Voltage_OpenWireSummary
+ * Runs the open wire method with print=true
```

Comment:
Since we're trying to abstract this out a bit, it might be better to include a high-level description of what we can expect the function to actually accomplish.